src/interface/src/app/map/plan-create-dialog/plan-create-dialog.component.html

```diff
@@ -0,0 +1,26 @@
+<h1 mat-dialog-title>Start a new plan in this planning area?</h1>
+<mat-dialog-content>
+    <mat-form-field [ngStyle]="{'min-width': '480px'}">
+      <mat-label>Plan name</mat-label>
+      <input matInput [formControl]="planNameControl">
+    </mat-form-field>
+  <p>
+    Once a plan is created, you can choose focuses & constraints, create scenarios,
+    and develop your proposal as well as share with collaborators.
+  </p>
+</mat-dialog-content>
+<mat-dialog-actions>
+  <button
+    mat-raised-button
+    class="cancel-button roundedButton"
```

Comment:
Where is the style for this (and submit-button)? The corresponding scss file seems to be empty.

Reply:
Removed "roundedButton" which was leftover. The buttons are using default material styling.